packages/wdio-browserstack-service/src/Percy/Percy.ts

```diff
@@ -0,0 +1,174 @@
+import fs from 'node:fs'
+import path from 'node:path'
+import os from 'node:os'
+
+import { createRequire } from 'node:module'
+const require = createRequire(import.meta.url)
+const { spawn } = require('node:child_process')
```

Comment:
We don't want to use `require` anymore
```suggestion
import { spawn } from 'node:child_process'
```

Reply:
Done